jerry-core/ecma/base/ecma-helpers-value.c

```diff
@@ -333,7 +379,24 @@ ecma_get_object_from_value (ecma_value_t value) /**< ecma-value */
 } /* ecma_get_object_from_value */
 
 /**
- * Copy ecma-value.
+ * Get the value from an error ecma value
+ *
+ * @return ecma value
+ */
+ecma_value_t __attr_pure___
+ecma_get_value_from_error_value (ecma_value_t value) /**< ecma value */
+{
+  JERRY_ASSERT (ecma_is_value_error (value));
+
+  value = (ecma_value_t) (value & ~(1u << ECMA_VALUE_ERROR_POS));
```

Comment:
We should use a common style for shifting operations (we are mixing 1, 1u, 1ull)